src/Model/MyParcelRequest.php

```diff
@@ -167,7 +168,13 @@ public function sendRequest($method = 'POST', $uri = self::REQUEST_TYPE_SHIPMENT
         $request->close();
 
         if ($this->getError()) {
-            throw new ApiException('Error in MyParcel API request: ' . $this->getError() . ' Url: ' . $url . ' Request: ' . $this->body);
+            switch ($this->errorCode->first()) {
```

Comment:
Kan je hier wel `->first()`doen? Is dit een collection? Volgens mij is dit een array en kan je `Arr::first($this->errorCodes)` gebruiken.

Reply:
O, ik zie nu ook een andere comment van @RichardPerdaan dat gaat over het zelfde.